server/src/test/java/com/google/coffeehouse/common/PersonTest.java

```diff
@@ -0,0 +1,104 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.coffeehouse.common;
+
+import com.google.coffeehouse.util.IdentifierGenerator;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+
+/**
+ * Unit tests for {@link Person}.
+ */
+@RunWith(JUnit4.class)
+public final class PersonTest {
+  private static final String NICKNAME = "Arthi";
+  private static final String ALT_NICKNAME = "New Name";
+  private static final String EMAIL = "test@fake.fake";
+  private static final String ALT_EMAIL = "New Email";
+  private static final String PRONOUNS = "she/her";
+  private static final String ALT_PRONOUNS = "New Pronouns";
+  private static final String IDENTIFICATION_STRING = "predetermined-identification-string";
+  private Person.Builder personBuilder;
+  
+  @Mock private IdentifierGenerator idGen;
+
+  @Before
+  public void beforeTest() {
```

Comment:
This is just a nitpick but do you think we could change this to be setUp() instead? I think it is Google convention if I'm not mistaken.

Reply:
Very valid point, consistency is important and I have changed the name of the functions :)